### ISSUE TITLE
Can I choose additional parameters to specify my training(not epochs but achieve particular loss value)?

### ISSUE THREAD

**Issue by Oysiyl** [minimaxir/textgenrnn#61]
Hi @minimaxir! I used you library and it`s awesome! But I didn`t find in docs information about how to choose not number of epochs but specify certain value of loss. 
For example, I just want to get a model that will be good to predict. I have 584,000 character sequences and I want to get a good model.
And another question - with what value loss can the model be considered good (on a similar volume of text)?
It would be great to be able to ignore the number of epochs and just put the parameter (such as def_loss = 0.01) and model  will train until loss == 0.01.

**Comment by minimaxir** [comment]
No, for a few reasons:

1) Models aren't guaranteed to get a specific training loss no matter how long you train it: it depends on architecture and the data.

2) A loss that's super low means the model overfits, which is bad for making creative results.

The best way to get better models is to train on more data; in lieu of that, you can train longer and set the `dropout` parameter to `0.1` or `0.2`.
